liquibase-standard/src/test/java/liquibase/database/core/OracleDatabaseTest.java

```diff
@@ -151,5 +152,89 @@ public void getDateLiteral_datetime_invalid() {
     public void getDateLiteral_unsupported() {
         assertEquals("UNSUPPORTED:123", database.getDateLiteral("123"));
     }
+
+    @Test
```

Comment:
Hi @m4tt30c91,

We very much appreciate you adding several scenarios to test this change. One small suggestion about it is instead of adding a few very similar tests, adding only one and parameterizing it would look better. To do you can add something like this:

```java
private static Stream<Arguments> method() {
        return Stream.of(
                Arguments.of("SOME_TABLE_WITH_MORE_THAN_30_CHARACTERS", "PK_SOME_TABLE_WITH_MORE_THAN_3", OracleDatabase.ORACLE_12C_MAJOR_VERSION, "Should truncate primary key to 30 characters"),
                ...
               Arguments.of()
        );
    }

    @ParameterizedTest
    @MethodSource("method")
    @Test
    public void shouldTruncatePrimaryKeyNameCorrectlyDependingOnDBVersion(String tableName, String expectedPrimaryKeyName, int databaseVersion, String assertMessage) throws Exception {
        final Database database = getDatabase();
        final Database spyDatabase = Mockito.spy(database);
        Mockito.when(spyDatabase.getDatabaseMajorVersion()).thenReturn(databaseVersion);
        Mockito.when(spyDatabase.getDatabaseMinorVersion()).thenReturn(1);
        String primaryKeyName = spyDatabase.generatePrimaryKeyName(tableName);
        assertEquals(expectedPrimaryKeyName, primaryKeyName, assertMessage);
    }
```

How does it sound to you?

Reply:
Hi @MalloD12 ,

Yeap thanks for the suggestion, I did a new commit to simplify the tests.
Let me know if that is ok :)